ros_buildfarm/templates/snippet/add_buildfarm_user.Dockerfile.em

```diff
@@ -0,0 +1,3 @@
+# Add user 'buildfarm', removing any existing user with that UID
+RUN if [ $(id -nu @(uid)) ]; then userdel -r $(id -nu @(uid)); fi
```

Comment:
```suggestion
RUN if [ $(id -nu @(uid)) ]; then userdel -r $(id -nu @(uid)); fi && useradd -u @(uid) -l -m buildfarm
```

I would suggest making this a single RUN command to avoid the extra layer in between which no one will actually want to use.  And delete the next line.